dynamorestart/DynamoRestart.php

```diff
@@ -0,0 +1,131 @@
+                if (!str_contains($e->getMessage(), "400 Bad Request")) {
+                    throw $e;
+                }
+            }
+        }
+
+        print_r("Creating dynamodb tables." . PHP_EOL);
+
+        $this->client->createTable([
+            'TableName' => WebauthnTable,
+            'KeySchema' => [['AttributeName' => 'uuid', 'KeyType' => 'HASH']],
+            'AttributeDefinitions' => [['AttributeName' => 'uuid', 'AttributeType' => 'S']],
+            'ProvisionedThroughput' => ['ReadCapacityUnits' => 10, 'WriteCapacityUnits' => 10],
+        ]);
+
+        $this->client->createTable([
+            'TableName' => ApiKeyTable,
+            'KeySchema' => [['AttributeName' => 'value', 'KeyType' => 'HASH']],
+            'AttributeDefinitions' => [['AttributeName' => 'value', 'AttributeType' => 'S']],
+            'ProvisionedThroughput' => ['ReadCapacityUnits' => 10, 'WriteCapacityUnits' => 10],
+        ]);
+
+        $this->client->waitUntil('TableExists', array(
+            'TableName' => ApiKeyTable
+        ));
+        print_r("Finished creating dynamodb tables." . PHP_EOL);
+    }
+
+    public function initApiKeys()
+    {
+        print_r("Creating api key(s)." . PHP_EOL);
+
+        $this->client->putItem([
+            'Item' => [
+                'value' => [
+                    'S' => ApiKeyValue,
+                ],
+                'hashedApiSecret' => [
+                    'S' => '$2a$10$8Bp9PqqfStjLvh1nQJ67JeY3CO/mEXmF1GKfe8Vk0kue1.i7fa2mC',
+                ],
+                'email' => [
+                    'S' => 'example-user@example.com',
+                ],
+                'activatedAt' => [
+                    'N' => '1590518080000',
+                ],
+                'createdAt' => [
+                    'N' => '1590518080000',
+                ],
+            ],
+            'TableName' => ApiKeyTable,
+        ]);
+    }
+
+    public function initWebauthnEntries()
+    {
+        print_r("Creating WebauthnEntries." . PHP_EOL);
+
+        $this->client->putItem([
+            'Item' => [
+                'uuid' => [
+                    'S' => '097791bf-2385-4ab4-8b06-14561a338d8e',
+                ],
+                'apiKey' => [
+                    'S' => ApiKeyValue,
+                ],
+                'encryptedAppId' => [
+                    'S' => 'SomeEncryptedAppId',
+                ],
+                'encryptedKeyHandle' => [
+                    'S' => 'SomeEncryptedKeyHandle',
+                ],
+//                'encryptedCredentials' => [
+            //  This doesn't work. What binary representation would work here?
+//                    'B' => 'ed634d2c138412f0e5d0f85ac8bceac9264df24a0bf597e75038caf9bb7cb6363beb7b8e9c660475b730fa4f29222b481cc76231d79ea8f8e8a4b0b2ebca3c315e9309db62c07ef0d4264073f1f6741b600086af6fa2d8657f660a1d415fc65ac907e2828865940fe2bfcc977577df1b35463dd04432dc2a746ca712e326ede06e3fa9d72f0a274d',
+//                ],
```

Comment:
You might be able to use `hex2bin()`, though I don't know for sure:
```suggestion
                'encryptedCredentials' => [
                    'B' => hex2bin('ed634d2c138412f0e5d0f85ac8bceac9264df24a0bf597e75038caf9bb7cb6363beb7b8e9c660475b730fa4f29222b481cc76231d79ea8f8e8a4b0b2ebca3c315e9309db62c07ef0d4264073f1f6741b600086af6fa2d8657f660a1d415fc65ac907e2828865940fe2bfcc977577df1b35463dd04432dc2a746ca712e326ede06e3fa9d72f0a274d'),
                ],
```

Reply:
Thanks. That worked!